scripts/find.rb

```diff
@@ -0,0 +1,2 @@
+People.where(('age > :age', age: 20 ) || title: 'Queen of Meereen and Mother of Dragons')
```

Comment:
There's two issues with this code:

1) The class name should be singular i.e. `Person.where`

2) The syntax for the SQL fragments that you use in your query is off. Try something like:

`Person.where('age > :year or title = :title', year: 40, title: 'Queen of Meereen and Mother of Dragons')`

Query logic first, then map your symbols to values on the right